style.css

```diff
@@ -278,3 +282,205 @@ footer {
   gap: 20px;
   font-size: 22px;
 }
+
+.hide-mobile {
+  display: none;
+}
+
+.hide-desktop {
+  display: block;
+}
+
+.fa-envelope {
+  display: none !important;
+}
+
+@media screen and (min-width: 768px) {
+  .fa-bars {
+    display: none !important;
+  }
+
+  .fa-envelope {
+    display: block !important;
+  }
+
+  .hide-mobile {
+    display: block;
+  }
+
+  .hide-desktop {
+    display: none;
+  }
+
+  nav {
+    display: flex;
+    gap: 20px;
+  }
+
+  header {
+    padding: 20px 70px;
+  }
```

Comment:
- [ ] Kindly ensure your logo and all navbar items have the correct font-family and size as in the figma design